hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -15121,26 +15108,8 @@ e_byzantium = {
                     venetian = "Aitos" # Historical? Copied from Italian
                 }
                 b_yambol = {
-                    bosnian = "Jambol"
-                    bulgarian = "Jambol"
-                    carantanian = "Jambol"
-                    croatian = "Jambol"
-                    polish = "Jambol"
-                    pommeranian = "Jambol" # Copied from Polish
-                    romanian = "Iambol"
```

Comment:
the romanian Iambol is correct, can you add it back?

Reply:
The reason I went with Iambol was that other languages in the base HIP mod used Yambol (or names based on Yambol, I don't remember) so I assumed it was still withing the medeival period.

I don't know for sure when the name Yambol first appeared.

Reply:
probably under the ottomans..
would Diampolis fit in romanian? or changed slightly somehow? Diambol?
I think romanian should generally (if possible) use names that are of pre-slavic/bulgar origin.
Otherwise you would have to take the slavicized Dbilin and romanize it again somehow.

Reply:
so
greek = "Hiambouli"
roman = "Diampolis"
is definite and looking at 
roman = "Sympheropolis"
romanian = "Simferopol"

i suggest adding romanian = "Diampol"